### PR TITLE
release page link 404

### PR DESCRIPTION
**Pull request by rainmor**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:
change release pages
e.g https://www.ovirt.org/release/4.1.4/Hosted_Engine_Howto#release 4.1.4 Hosted Engine Howto
the page is not exist
-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/develop/infra/infrastructure.html.md [hunk 1]
```diff
@@ -5,17 +5,17 @@ authors: dcaroest, dneary, eedri, ekohl, eyale, herrold, knesenko, ngoldin, quai
   rmiddle
 ---
 
-# Infrastructure
+# Infrastructure 基础设施
```
Comment (mykaul) [inline source/develop/infra/infrastructure.html.md]: Which language is it translated to? I wonder if we should have a page per langugage.

Comment (duck-rh) [reply source/develop/infra/infrastructure.html.md]: It is Chinese.

I don't see any page translated in other languages currently but that would be nice.

File: setup.sh [hunk 2]
```diff
@@ -15,5 +15,6 @@ else
     echo "Could not verify system is RedHat or Debian."
     exit 1
 fi 
-
+gem sources --add http://gems.ruby-china.org/ --remove https://rubygems.org/
```
Comment (duck-rh) [inline setup.sh]: Why push these changes?
That's probably nice for Chinese contributors, but not for the whole world.

Comment (JohnMarksRH) [reply setup.sh]: looks weird to me too.

Comment (sandrobonazzola) [reply setup.sh]: I guess it's for speeding up local build in china. Please drop these changes from the pull request.

**Comment by sandrobonazzola** [comment]
Closing due to inactivity on change request for more than 2 months. Please send a new pull request if needed

**Pull request closed without merging**